### DISCUSSION TITLE
Using Tailwind CSS Styles in Markdown/MDX content w/@next/mdx?

### DISCUSSION THREAD

**Discussion by colinsteidtmann** [vercel/next.js#44252] [Help] [answered]
Hi, I have this in my `./page.mdx`
```
# Hello World
<p className="text-100-blue">This is blue text</p>
```
and I'm using tailwindcss, imported into the global style sheet, set it up as they instructed, etc. 

But the styles aren't being applied to my markdown content. I sense that there might be something limiting me, idk, is there anyway to get around this? I know I could wrap the content in a container with `className="prose"` from `@tailwindcss/typography` but then I'm kind of limited in my styling. 

Help would be kindly appreciated

**Comment by ciruz** [comment, marked as answer]
Hey,

can you show me your ``tailwind.config.js``?

You need to define the template paths there, basically tell tailwind where to look for your templates and css classes.

Checkout the template path configuration here (step 3): https://tailwindcss.com/docs/guides/nextjs

**Reply by colinsteidtmann** [reply]
Yeah, I found the problem. I changed my tailwind configuration from this:
```
module.exports = {
  (...)
  content: [
    "./pages/**/*.{js,ts,jsx,tsx,md, mdx}",
    "./components/**/*.{js,ts,jsx,tsx,md, mdx}",
    "./content/**/*.{js,ts,jsx,tsx,md, mdx}",
    (...)
};
```
to this:
```
module.exports = {
  (...)
  content: [
    "./pages/**/*.{js,ts,jsx,tsx,md,mdx}",
    "./components/**/*.{js,ts,jsx,tsx,md,mdx}",
    "./content/**/*.{js,ts,jsx,tsx,md,mdx}",
    (...)
};
```

A single space between the ',' and 'mdx' was breaking things 🤦‍♂️
